libV2/schemaUtils.js

```diff
@@ -2366,16 +2581,20 @@ module.exports = {
     let url = resolveUrlForPostmanRequest(path),
       baseUrlData = resolveBaseUrlForPostmanRequest(operationItem[method]),
       requestName = resolveNameForPostmanReqeust(context, operationItem[method], url),
-      queryParams = resolveQueryParamsForPostmanRequest(context, operationItem, method),
-      headers = resolveHeadersForPostmanRequest(context, operationItem, method),
-      pathParams = resolvePathParamsForPostmanRequest(context, operationItem, method),
+      { queryParamTypes, queryParams } = resolveQueryParamsForPostmanRequest(context, operationItem, method),
+      { headerTypes, headers } = resolveHeadersForPostmanRequest(context, operationItem, method),
+      { pathParamTypes, pathParams } = resolvePathParamsForPostmanRequest(context, operationItem, method),
       { pathVariables, collectionVariables } = filterCollectionAndPathVariables(url, pathParams),
       requestBody = resolveRequestBodyForPostmanRequest(context, operationItem[method]),
+      bodyTypes = context.resolvedSchemaTypes ? context.resolvedSchemaTypes[0] : {},
```

Comment:
We should not use `context` for these data, I feel we had discussed this already that we context is not the correct place to store these data.